src/agent/rustjail/src/container.rs

```diff
@@ -1390,13 +1388,13 @@ impl LinuxContainer {
         .context(format!("cannot change onwer of container {} root", id))?;
 
         if config.spec.is_none() {
-            return Err(nix::Error::Sys(Errno::EINVAL).into());
+            return Err(nix::Error::EINVAL.into());
         }
 
         let spec = config.spec.as_ref().unwrap();
 
         if spec.linux.is_none() {
-            return Err(nix::Error::Sys(Errno::EINVAL).into());
+            return Err(nix::Error::EINVAL.into());
```

Comment:
Great, the new version of `nix` also eliminates the problem of mixing `nix::Error::Sys()` and `nix::Error::from_errno()`.

@jodh-intel , do you think should we also unify how to return an `err`, now we have at least two styles(after your patch):

- `return Err(anyhow!(nix::Error::EINVAL));`
- `return Err(nix::Error::EINVAL.into());`

Reply:
Agreed. Which do you prefer?

Reply:
If have to choose one, I prefer the  `return Err(anyhow!(nix::Error::EINVAL));`.

Reply:
@liubin - branch updated.